lib/class-fm-demo-select.php

```diff
@@ -75,6 +75,14 @@ public function init() {
 		) );
 		$fm->add_meta_box( 'Repeatable Select Fields', 'demo-select' );
 
+		$fm = new Fieldmanager_Select( array(
+			'name'    => 'repeatable_multi_select',
+			'options' => array( 'One', 'Two', 'Three' ),
+			'limit' => 0,
+			'multiple' => true,
+		) );
+		$fm->add_meta_box( 'Repeatable Multiselect', 'demo-select' );
```

Comment:
I realize this is outside of the scope of this PR, but should these strings be getting localized? (I also realize this may be a part of a much larger undertaking.)